website/redirects.js

```diff
@@ -79,4 +79,14 @@ module.exports = [
       "https://www.github.com/firezone/firezone/releases/download/gateway-1.1.2/firezone-gateway_1.1.2_armv7",
     permanent: false,
   },
+  /*
+   * Redirects for old KB URLs
+   *
+   */
+  {
+    // TODO: Remove on or after 2024-10-21 after crawlers have re-indexed
+    source: "/kb/user-guides/:path",
+    destination: "/kb/client-apps/:path",
+    permanent: true,
+  },
```

Comment:
This alone doesn't cover `﻿﻿/kb/user-guides` with no trailing slash, which is the 2nd result on DuckDuckGo for "firezone user guides"

```suggestion
  {
    // TODO: Remove on or after 2024-10-21 after crawlers have re-indexed
    source: "/kb/user-guides/:path",
    destination: "/kb/client-apps/:path",
    permanent: true,
  },
  {
    source: "/kb/user-guides",
    destination: "/kb/client-apps",
    permanent: true,
  },
```

Reply:
Good catch I'll see if removing the last slash will achieve both

Reply:
didn't look like it